client/src/components/Dashboard-FormAssemply.style.js

```diff
@@ -0,0 +1,27 @@
+import styled, { css } from 'styled-components';
+
+export const FormWrapper = styled.section`
+  border-top: 3px solid var(--black);
+  border-bottom: 3px solid var(--black);
+  box-sizing: border-box;
+  margin-top:1.5rem;
+`;
+export const Form = styled.div`
```

Comment:
could this be a ```styled.form``` ?

Reply:
oh no , it's not a form , it just the name

Reply:
In that case it might be better to find another name for it, because calling it Form can be a bit confusing. For example ```FormLink``` or ```FormSection```